odl/discr/lp_discr.py

```diff
@@ -211,10 +211,15 @@ def is_uniform_byaxis(self):
 
     @property
     def is_uniform(self):
-        """``True`` if ``self.partition`` is uniform."""
+        """``True`` if `partition` is uniform."""
         return self.partition.is_uniform
 
     @property
+    def has_isotropic_cells(self):
```

Comment:
Not quite sure if we need this passthrough, accessing the partition directly should be ok